Examples/BookingSystem.AspNetFramework/BookingSystem.AspNetFramework.csproj

```diff
@@ -113,10 +113,6 @@
       <HintPath>..\..\packages\Microsoft.AspNetCore.WebUtilities.2.1.0\lib\netstandard2.0\Microsoft.AspNetCore.WebUtilities.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.Data.Sqlite, Version=3.1.4.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
-      <HintPath>..\..\packages\Microsoft.Data.Sqlite.Core.3.1.4\lib\netstandard2.0\Microsoft.Data.Sqlite.dll</HintPath>
```

Comment:
Note removed Microsoft.Data.Sqlite due to installation issues with .NET Framework (https://github.com/ericsink/SQLitePCL.raw#how-does-this-compare-to-microsoftdatasqlite, SQLitePCL.raw does not work out-of-the-box on Windows with .NET Framework). Replaced with System.Data.SQLite